vumi_message_store/api/tests/test_message_export_worker.py

```diff
@@ -0,0 +1,494 @@
+# -*- coding: utf-8 -*-
+
+import json
+from datetime import datetime
+from urllib import urlencode
+
+from twisted.internet.defer import inlineCallbacks, returnValue
+from twisted.web import http
+
+from vumi_message_store.message_store import (
+    MessageStoreBatchManager, OperationalMessageStore)
+from vumi_message_store.api.message_export_worker import MessageExportWorker
+
+from vumi.utils import http_request_full
+
+from vumi.tests.helpers import (
+    VumiTestCase, MessageHelper, PersistenceHelper, WorkerHelper)
+
+
+class TestMessageExportWorker(VumiTestCase):
+
+    @inlineCallbacks
+    def setUp(self):
+        self.persistence_helper = self.add_helper(
+            PersistenceHelper(use_riak=True))
+        self.worker_helper = self.add_helper(WorkerHelper())
+        self.msg_helper = self.add_helper(MessageHelper())
+
+        riak, redis = yield self.create_managers()
+        self.operational_store = OperationalMessageStore(riak, redis)
+        self.batch_manager = MessageStoreBatchManager(riak, redis)
+
+    @inlineCallbacks
+    def create_managers(self):
+        riak = yield self.persistence_helper.get_riak_manager()
+        redis = yield self.persistence_helper.get_redis_manager()
+        self.add_cleanup(self.close_managers, riak, redis)
+        returnValue((riak, redis))
```

Comment:
Instead of a separate `close_managers` method that closes both at once, why not just `self.add_cleanup(thing.close_manager)` when you create each manager?